tests/tck/features/schema/Schema.feature

```diff
@@ -399,9 +402,8 @@ Feature: Insert string vid of vertex and edge
       CREATE TAG animal(name string, kind string);
       """
     Then the execution should be successful
-    Given wait 3 seconds
     # check result
-    When executing query:
+    When try to execute query:
```

Comment:
here not need `try`

Reply:
Will the `DESCRIBE` command ask the metad the schema information ?

Reply:
Yes

Reply:
fixed